api/v1alpha1/smbshare_types.go

```diff
@@ -117,6 +117,10 @@ type SmbShareStatus struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 
+// +kubebuilder:printcolumn:JSONPath=`.spec.shareName`,description="Name of the Samba share",name="Share-name",type=string
+// +kubebuilder:printcolumn:JSONPath=`.spec.storage.pvc.path`,description="Connect path for the share within PVC",name="Share-path",type=string
```

Comment:
I think we can lose the word "Connect" from "Connect path" without losing any meaning. What do you think?

Reply:
Fine.